src/SWP/Bundle/AnalyticsBundle/Repository/RequestMetricRepository.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Content Bundle.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\AnalyticsBundle\Repository;
+
+use SWP\Bundle\AnalyticsBundle\Model\RequestMetric;
+
+class RequestMetricRepository extends \Doctrine\ORM\EntityRepository
+{
+    /**
+     * @param RequestMetric $requestMetric Persists object and flushes the entity manager
+     */
+    public function save(RequestMetric $requestMetric)
```

Comment:
I would name that method `add` as it's better reflecting the operation you want to do, `save` would mean you want to update the resource and the repository should not be capable of that.